packages/lib/src/components/helpers/QRLoaderContainer.tsx

```diff
@@ -49,6 +50,10 @@ class QRLoaderContainer<T extends QRLoaderContainerProps = QRLoaderContainerProp
         };
     }
 
+    protected submitAnalytics = (aObj: SendAnalyticsObject) => {
```

Comment:
I think we can remove the override `submitAnalytics` since it doesn't have other logic except for calling the parent one.

Reply:
yup